src/core_modules/capture-core/components/Pages/ViewEvent/epics/viewEvent.epics.js

```diff
@@ -22,7 +22,7 @@ import {
     openViewEventPageFailed,
     initializeWorkingListsOnBackToMainPage,
 } from '../ViewEventComponent/viewEvent.actions';
-import { actionTypes as eventListActionTypes } from '../../MainPage/EventsList/eventsList.actions';
+import { listViewActionTypes as eventListActionTypes } from '../../../ListView';
```

Comment:
Repeating myself on this one. Maybe we can avoid the use of as here now that the action types have such a good distinct name ;)

Reply:
Removed in later PR.